taskManager/app/controllers/tasks_controller.rb

```diff
@@ -2,14 +2,14 @@ class TasksController < ApplicationController
   before_action :task, only: %i[destroy show edit update]
 
   ORDER = %w[asc desc].freeze
-  PER = 5,freeze
+  ROWS_PER_PAGE = 5.freeze
```

Comment:
```suggestion
  ROWS_PER_PAGE = 5
```
定数が数字の場合は、`freeze`は不要です！

Reply:
ぬおお、そうなんですね。